src/components/accession2/edit/Accession2EditModal.tsx

```diff
@@ -82,7 +82,13 @@ export default function Accession2EditModal(props: Accession2EditModalProps): JS
             readonly={true}
           />
         </Grid>
-        <Species2Dropdown record={record} organization={organization} setRecord={setRecord} validate={validateFields} />
+        <Species2Dropdown
+          speciesId={record.speciesId}
```

Comment:
Ups, that was me. Thanks!

Reply:
no worries, maybe i should have named the property as 'initialValue' or something.